.github/workflows/testing.yml

```diff
@@ -8,14 +12,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.8", "3.11"]
+        os: ["ubuntu-latest", "macos-13", "windows-latest"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Action runners are easy to come by nowadays, and our suite is so light that I think it's fine to test against all versions